react-payments-app/src/components/Card/CardShapedButton/card.stories.jsx

```diff
@@ -0,0 +1,11 @@
+import CardShapedButton from "./Card";
+import "../card.css";
+
+export default {
+  title: "Components/Card/CardShapedButton",
+  component: CardShapedButton,
+};
+
+const Template = (args) => <CardShapedButton {...args} />;
+
+export const Default = Template.bind({});
```

Comment:
```suggestion
import CardShapedButtonComponent from "./card";
import "../card.css";

export default {
  title: "Components/Card/CardShapedButton",
  component: CardShapedButtonComponent,
};

const Template = (args) => <CardShapedButton {...args} />;

export const CardShapedButton = Template.bind({});
```

(소소)

오직 하나의 스토리만 있고, 스토리명이 컴포넌트명과 일치한다면 아래와 같이 depth 없이 바로 나타낼 수 있습니다 😄 

![image](https://user-images.githubusercontent.com/31029000/219870947-e746eb0f-82e8-4c48-9163-f6cdaea639b5.png)


https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy#single-story-hoisting

Reply:
오...! 꿀팁 감사합니다 😁